tests/phpunit/elementor/data/base/test-standards.php

```diff
@@ -0,0 +1,372 @@
+<?php
+namespace Elementor\Tests\Phpunit\Elementor\Data\Base;
+
+/*
+ * The tests aim to cover the rest standards for api routing running the endpoints via `Data\Manager::run`.
+ * The deepest level that should be validated in each test is '/alpha/1/beta/1/gamma/1/'.
+ *
+ * Try reach the standards with controllers, sub-controllers, endpoints, sub-endpoints.
+ * at least show where the limits for them.
+ */
+
+use Elementor\Tests\Phpunit\Elementor\Data\Base\Mock\Standards\OnlyController\AlphaController as OnlyControllerAlphaController;
+use Elementor\Tests\Phpunit\Elementor\Data\Base\Mock\Standards\OnlySubController\BetaController as OnlySubControllerBetaController;
+use Elementor\Tests\Phpunit\Elementor\Data\Base\Mock\Standards\OnlyEndpoint\GammaEndpoint as OnlyEndpointGammaEndpoint;
+use Elementor\Tests\Phpunit\Elementor\Data\Base\Mock\Standards\OnlySubEndpoint\BetaSubEndpoint as OnlySubEndpointBetaSubEndpoint;
+use Elementor\Tests\Phpunit\Elementor\Data\Base\Mock\Standards\OnlySubEndpoint\GammaSubEndpoint as OnlySubEndpointGammaSubEndpoint;
```

Comment:
```suggestion
use Elementor\Tests\Phpunit\Elementor\Data\Base\Mock\Standards\OnlyController;

Reply:
@matipojo, what do you mean?

Reply:
Shorten

Reply:
Handled: https://github.com/elementor/elementor/pull/12820/commits/57310f6150c9f03cd50ce1c4c27799960a1a2088